CHANGELOG.md

```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+- Settings link appears in plugins page
+
+### Fixed
+- Lowercased company name in plugin metadata
+- Bug where preview pages had public cache headers set
```

Comment:
Nitpick: the bug could potentially have impacted any front-end post which should have had a `no-cache` header if developer mode was active. (e.g. on `main`, if I activate developer mode, I get public cache headers when viewing the front-end whilst logged in).